### PR TITLE
[vcpkg baseline] Fix baseline errors

### PR DESCRIPTION
**Pull request by JackBoosY**
- Since xmsh REPO is no longer available, set it to an empty port.

Related: #12063 #11637 #12068 etc.

### CODE REVIEW

**Comment by JackBoosY** [comment]
@BillyONeal And The `apr-util` build failed because port `apr` archive file _installed/x64-osx/build-1/apr_rules.mk_ cannot be found on the pipeline machine, so I think #11961 should be merged.

**Review by PhoebeHui — APPROVED** [review]

**Comment by BillyONeal** [comment]
Hmmm should we just delete the port outright rather than letting it successfully 'install' like this?

File: ports/xmsh/portfile.cmake [hunk 1]
```diff
@@ -1,3 +1,9 @@
+set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+
+message("xmsh REPO has been removed")
+
+return()
+
 vcpkg_find_acquire_program(PYTHON3)
```
Comment (vicroms) [inline ports/xmsh/portfile.cmake:7]: Please remove the unnecessary code in the portfile

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/xmsh/portfile.cmake (hunk 1 above)
Comment (JackBoosY) [reply ports/xmsh/portfile.cmake]: Removed.

**Comment by NancyLi1013** [comment]
Related issue #11923

**Comment by ras0219-msft** [comment]
>Since xmsh REPO is no longer available, set it to an empty port.

We should never just be replacing ports with "empty" -- if the port is replaced by something else, it should have a `Build-Depends:`. In this case, I believe the appropriate path is:

1. Mark it as skipped
2. If 6 months pass without a user submitting a fix, we can then remove it.

**Review by vicroms — APPROVED** [review]

**Comment by BillyONeal** [comment]
pixman:arm64-windows upstream server down during build
quickfix:arm64-windows Not sure exactly what happened, will merge this to nuke tombstone to try again:

```
D:\buildtrees\quickfix\src\v1.15.1-5086b0da20\src\C++\stdafx.h(14): warning C4005: '_WINSOCK_DEPRECATED_NO_WARNINGS': macro redefinition
D:\buildtrees\quickfix\src\v1.15.1-5086b0da20\src\C++\test\DictionaryTestCase.cpp: note: see previous definition of '_WINSOCK_DEPRECATED_NO_WARNINGS'
```

itp:x64-windows: upstream server down during build
fontconfig:x86-windows: upstream server down during build
rapidxml:x86-windows: upstream server down during build
itk:x64-osx:
```
Undefined symbols for architecture x86_64:
  "gdcm::Rescaler::SetMinMaxForPixelType(double, double)", referenced from:
      itk::GDCMImageIO::Write(void const*) in libITKIOGDCM-5.0.a(itkGDCMImageIO.cxx.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```

No idea here, it doesn't repro locally. Might be a race in itk's build itself.

fadbad:x64-linux: upstream server down during build.

As a result of the above investigation, merging this.

**Pull request merged**